docs/setup-vendor.md

```diff
@@ -0,0 +1,47 @@
+# Getting started with Private Packagist for Vendors in a Composer project
+
+Using Private Packagist for Vendors, customers install packages with Composer using a unique URL and an authentication token.
+
+Customers can start using Private Packagist in either an existing or a new Composer project.
```

Comment:
They don't want to start using Private Packagist, they want to start using the vendor's packages.